edge-modules/functions/binding/src/Microsoft.Azure.WebJobs.Extensions.EdgeHub/Microsoft.Azure.WebJobs.Extensions.EdgeHub.csproj

```diff
@@ -14,14 +14,19 @@
     <Title>Microsoft Azure Edge Function extension</Title>
     <Authors>Microsoft</Authors>
     <PackageRequireLicenseAcceptance>true</PackageRequireLicenseAcceptance>
-    <PackageLicenseUrl>https://github.com/Azure/iotedge/blob/master/LICENSE</PackageLicenseUrl>
+    <PackageLicenseFile>LICENSE</PackageLicenseFile>
     <Description>This package contains binding extensions for IoTEdge.</Description>
-    <PackageIconUrl>http://go.microsoft.com/fwlink/?LinkID=288890</PackageIconUrl>
+    <PackageIcon>nugetIcon.png</PackageIcon>
     <PackageProjectUrl>https://github.com/Azure/iotedge</PackageProjectUrl>
     <Copyright>© Microsoft Corporation. All rights reserved.</Copyright>
     <PackageTags>IoT Microsoft Azure IoTEdge Functions</PackageTags>
   </PropertyGroup>
 
+  <ItemGroup>
+      <None Include="..\..\..\..\..\LICENSE" Pack="true" PackagePath=""/>
+      <!-- Orinal Icon: http://go.microsoft.com/fwlink/?LinkID=288890 -->
+      <None Include="..\..\..\..\..\doc\images\nugetIcon.png" Pack="true" PackagePath=""/>
```

Comment:
I didn't find nugetIcon.png based on given path.

Reply:
I just added them in this PR.